packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -662,6 +662,11 @@ export default class InteractSubmode extends Component<Signature> {
                 class={{cn
                   'operator-mode-stack'
                   (if backgroundImageURLSpecificToThisStack 'with-bg-image')
+                  (if
+                    (and (gt stack.length 1) (lt stack.length 3))
+                    'medium-padding-top'
+                  )
+                  (if (gt stack.length 2) 'small-padding-top')
```

Comment:
Perhaps it would it be more elegant to do this with CSS (`nth-child`)?

Reply:
Good point @lukemelia , I try to find ways can be done with `nth-child` but found out less likely possible as these padding-top classes are targeting in container level instead of childrens (.item.buried).

<img width="1412" alt="Screenshot 2024-10-08 at 1 56 45 PM" src="https://github.com/user-attachments/assets/7e3f791d-6deb-4a28-b12d-1eb627c9c909">